### PR TITLE
[CT- 1273] Add camera Resolution and FrameRate to the publisher

### PR DESCRIPTION
**Pull request by aacotroneo**

### CODE REVIEW

File: RCTOpenTokPublisherView.m [hunk 1]
```diff
@@ -55,8 +55,16 @@ - (void)mount {
  * inside the mounted view
  */
 - (void)startPublishing {
-    _publisher = [[OTPublisher alloc] initWithDelegate:self];
-
+    _publisher = [[OTPublisher alloc] initWithDelegate:self
+                                                  name: @"My Video"
+                                      cameraResolution: _cameraResolution
+                                       cameraFrameRate: (OTCameraCaptureResolution)_cameraFrameRate];
```
Comment (k7k0) [inline RCTOpenTokPublisherView.m]: bad casting? (resolution to frame rate?)

Comment (aacotroneo) [reply RCTOpenTokPublisherView.m]: ups!! this happens to me when things don't work!! I just modify things to see if something changes, it does not.. then I commit and push 😁

File: RCTOpenTokPublisherView.h [hunk 2]
```diff
@@ -15,6 +15,8 @@
 @property (nonatomic, strong) NSString *apiKey;
 @property (nonatomic, strong) NSString *sessionId;
 @property (nonatomic, strong) NSString *token;
+@property (nonatomic, assign) NSInteger cameraResolution;
+@property (nonatomic, assign) NSInteger cameraFrameRate;
```
Comment (tysonxor) [inline RCTOpenTokPublisherView.h:19]: I think these need to be pointers: 
NSInteger *cameraResolution;
NSInteger *cameraFrameRate;

Comment (tysonxor) [reply RCTOpenTokPublisherView.h:19]: Nevermind, NSInteger is considered a primitive type so using a pointer is not necessary

Comment (aacotroneo) [reply RCTOpenTokPublisherView.h:19]: cool! good to know.. I just assumed it was a primitive.. didnt really check!

File: RCTOpenTokPublisherView.m [hunk 3]
```diff
@@ -21,6 +21,9 @@ @implementation RCTOpenTokPublisherView {
     OTPublisher *_publisher;
 }
 
++ (void)sayHello{
+    NSLog(@"hello world");
+}
```
Comment (aacotroneo) [inline RCTOpenTokPublisherView.m:26]: this will turn into something like 'pause'.. have imagination!

**Pull request merged**